pkg/core/attributes/element.go

```diff
@@ -17,6 +17,7 @@ const (
 	Hydro
 	Dendro
 	Quicken // or overdose
+	Burning
```

Comment:
I don't believe Burning is an actual element but a reaction so we shouldn't add a new element here. See my comment in reactable for possible solution